src/models/decorators.ts

```diff
@@ -66,10 +67,50 @@ export function isColumnNullable(targetClass: Function, propertyName: string): b
     return false;
 }
 
```

Comment:
It looks like there are a couple of places where you've written JS Docs above the function. Is there a reason you didn't include them here?